Dockerfile

```diff
@@ -1,32 +1,12 @@
-ARG PYTHON_VER
+ARG PYTHON_VER=3.11
 
 FROM python:${PYTHON_VER}-slim
 
-# Install all OS package upgrades and dependencies needed to run Nautobot in production
-# hadolint ignore=DL3005,DL3008,DL3013
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install --no-install-recommends -y git mime-support curl libxml2 libmariadb3 openssl && \
-    apt-get autoremove -y && \
-    apt-get clean all && \
-    rm -rf /var/lib/apt/lists/* && \
-    pip --no-cache-dir install --upgrade pip wheel
-
-RUN pip install --upgrade pip
-
-RUN curl -sSL https://install.python-poetry.org -o /tmp/install-poetry.py && \
-    python /tmp/install-poetry.py --version 1.6.0 && \
-    rm -f /tmp/install-poetry.py
-
-# Add poetry install location to the $PATH
-ENV PATH="${PATH}:/root/.local/bin"
+RUN pip install --upgrade pip \
+  && pip install poetry
```

Comment:
I think our dev standards still prefers the installer script over the pip installer? I may be wrong.

Reply:
updated.

Reply:
fixed